src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+
+
+const ChatLog = (props) => {
+   
+    const getMessage = props.entries.map((message) =>{
+        
+        return (
+            <ChatEntry id={message.id} sender = {message.sender} key = {message.id}
+            body = {message.body} timeStamp = {message.timeStamp } liked ={message.liked} onLiked = {props.likedFunc} />
+        )
+
+    })
+    return <div className='chat-log'>{getMessage}</div>
```

Comment:
Also, instead of writing a method `getMessage` to reference the return value from calling map() on `props.entries`, it's more common to directly iterate over the chat entries in your component like this without using a variable: 
```javascript
const ChatLog = (props) => {
    return (
     <div className='chat-log'>
        {props.entries.map((message) => (
            <ChatEntry 
                key={message.id} 
                id={message.id}
                sender={message.sender}
                body={message.body}
                timeStamp={message.timeStamp}
                liked={message.liked}
                onLiked={props.likedFunc}
            />
        ))}
    </div>
    )
};